src/live/embed.rs

```diff
@@ -448,13 +455,15 @@ impl KargEmbedAreas {
             default,
             regions,
             args,
+            arch,
         })
     }
 
     fn parse(region: &Region) -> Result<String> {
         Ok(String::from_utf8(region.contents.clone())
             .context("invalid UTF-8 in karg area")?
             .trim_end_matches('#')
+            .trim_end_matches('\0')
```

Comment:
dumb question.. why do we need to use `\0` on s390x? I feel like we could add some comments on this somewhere (maybe not here, but when `placeholder` is set below.

Reply:
I don't know and will ask around. using `#` breaks the boot

Reply:
Thanks!

Reply:
Found only some s390x assembler and C code, where `0` is used as filler.